README.md

```diff
@@ -5,7 +5,38 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![Join the chat at https://gitter.im/poetapp/Lobby](https://badges.gitter.im/poetapp/Lobby.svg)](https://gitter.im/poetapp/Lobby)
 
-Po.et JS is a small library that provides methods to easily create and sign Po.et Claims. 
+Po.et JS is a small library that provides methods to easily create and sign Po.et Claims according to the 
+[Verifiable Credentials Data Model](https://w3c.github.io/vc-data-model). These claims are [JSON-LD](https://w3c.github.io/json-ld-syntax/)
+documents. As such, you can define your own JSON-LD `@context` to map your submitted Claims.
+
+Po.et does provide a default `@context object` that you can extend or override in the `createClaim` function. The current defaults are as follows:
+
+```ts  
+'@context': {
+   cred: 'https://w3id.org/credentials#',
```

Comment:
This link to credentials doesn't work: https://w3id.org/credentials#

but I noticed the security one does: https://w3id.org/security#

Not sure if the credentials one is incorrect or if there is just no redirect in place yet that soon will be?

Reply:
I do believe it is the latter.

Reply:
Actually at one point, it redirected to https://web-payments.org/contexts/credentials-v1.jsonld

Reply:
Some of these specs are currently passing from a committee to a formal W3 working group. So very much in flux atm

Reply:
Ok, thanks!